embedding.py

```
@@ -0,0 +1,56 @@
+import csv
+from sentence_transformers import SentenceTransformer
+import torch
+
+
+def combine_2_embeddings(embd_type, text, title, i, incongruent_paragraphs):
+    if embd_type == "concatenate":
+        x = torch.cat(text, title)
+    if embd_type == "subtract":
+        x = torch.sub(text, title)
```

If `emb_type` is neither `concatenate` nor `subtract`, the variable x won't be initialized. The function should raise `ValueError` with appropriate message, when `embd_type` is not specified correctly.